ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh

```diff
@@ -116,6 +116,10 @@ PRIVATE_SUBNET_1="$(echo "${PRIVATE_SUBNETS}" | sed 's/"//g' | cut -d, -f2)"
 PRIVATE_SUBNET_2="$(echo "${PRIVATE_SUBNETS}" | sed 's/"//g' | cut -d, -f3)"
 PUBLIC_SUBNETS="$(echo "${VPC_JSON}" | jq '.[] | select(.OutputKey == "PublicSubnetIds").OutputValue')"  # explicitly keeping wrapping quotes
 
+# Adapt step aws-provision-tags-for-byo-vpc, which is required by Ingress operator testing.
```

Comment:
@yunjiang29 could you help reminding me the background of this?
Adding tags to the `VPC` and `private subnets` is required for a normal UPI AWS install or just for our E2E testing?

Reply:
@gpei for out e2e test only, see https://github.com/openshift/release/pull/32597

Reply:
got it, thanks